### DISCUSSION TITLE
aria2 显示已推送其实没推过去，是不是还不支持 wss 协议呀？

### DISCUSSION THREAD

**Discussion by ricky0829** [AlistGo/alist#1337] [Q&A]
如题，我本人在外面，aria2 和 alist 部署在同一个服务器的不同容器里，我其实不知道该配服务器本地地址还是外部地址，所以服务器本地 IP 和暴露到外网的 wss 我都试了 ，都显示已推送，但从 ariang 里看不到任务（ariang 可以连接到 aria2 rpc），其实没有推过去，通过 SSH 上服务器看了下载目录也确实没有文件。

![image](https://user-images.githubusercontent.com/28993319/178959468-51cf77bd-69c4-40cc-be86-aba87d2cad7b.png)
![image](https://user-images.githubusercontent.com/28993319/178960193-90a96155-6172-4683-8942-6cf304fae81d.png)

**Comment by Windman1320** [comment]
你可以先看下这个地址是否能正常工作，有个方法是Chrome装个Aria2 for Chrome，网上随便找个链接send to看看能否发送过去
